.azure/scripts/setup-kind.sh

```diff
@@ -3,7 +3,10 @@ set -xe
 
 rm -rf ~/.kube
 
-KUBE_VERSION=${KUBE_VERSION:-1.25.0}
+# There is a bug in 0.24.0 - https://github.com/kubernetes-sigs/kind/issues/3713
```

Comment:
Can you follow up on https://github.com/kubernetes-sigs/kind/issues/3713 or file a new issue with details?

We're pretty sure there is NOT a bug and instead this is due to network policies installed by the user https://github.com/kubernetes-sigs/kind/issues/3713#issuecomment-2310412867

kind v0.24 enables networkpolicy support, as noted in the release notes, and that is NOT a bug, but it is a breaking change if you were installing network policies that break your traffic because previously they were not implemented.